--- a/modules/contrib/field_group/contrib/field_group_migrate/src/Plugin/migrate/source/d7/FieldGroup.php
+++ b/modules/contrib/field_group/contrib/field_group_migrate/src/Plugin/migrate/source/d7/FieldGroup.php
@@ -15,6 +15,8 @@
  *
  * @MigrateSource(
  *   id = "d7_field_group"
+ *   source_module = "field_group",
+ *   destination_module = "field_group"
  * )
  */
 class FieldGroup extends DrupalSqlBase {
