diff --git a/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php b/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
--- a/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
+++ b/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
@@ -42,7 +42,7 @@
 
     foreach ($files as $delta => $file) {
       $elements[$delta] = [
-        '#markup' => \Drupal\Component\Utility\DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', fn() => \Drupal\Core\StringTranslation\ByteSizeMarkup::create($file->getSize(), $file->language()->getId()), fn() => format_size($file->getSize(), $file->language()->getId())),
+        '#markup' => \Drupal\Component\Utility\DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', fn() => \Drupal\Core\StringTranslation\ByteSizeMarkup::create($file->getSize() ?? 0, $file->language()->getId()), fn() => format_size($file->getSize() ?? 0, $file->language()->getId())),
       ];
     }
 
