diff --git a/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php b/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
--- a/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
+++ b/src/Plugin/Field/FieldFormatter/FileSizeFormatter.php
@@ -42,7 +42,7 @@
 
     foreach ($files as $delta => $file) {
       $elements[$delta] = [
-        '#markup' => format_size($file->getSize(), $file->language()->getId()),
+        '#markup' => format_size($file->getSize() ?? 0, $file->language()->getId()),
       ];
     }
 
