--- a/src/Plugin/migrate/EntityReferenceTranslationDeriver.php	(date 1564448937996)
+++ b/src/Plugin/migrate/EntityReferenceTranslationDeriver.php	(date 1564448937996)
@@ -98,6 +98,9 @@
       foreach ($fields as $field_name => $field) {
         foreach ($field['bundles'] as $bundle) {
           $field_definitions = $this->entityFieldManager->getFieldDefinitions($entity_type, $bundle);
+          if (!isset($field_definitions[$field_name])) {
+            continue;
+          }
           $target_type = $field_definitions[$field_name]->getSetting('target_type');
 
           // If the field's target type is not supported, skip it.
