--- a/modules/contrib/imagemagick/src/Plugin/ImageToolkit/ImagemagickToolkit.php
+++ b/modules/contrib/imagemagick/src/Plugin/ImageToolkit/ImagemagickToolkit.php
@@ -1241,11 +1241,18 @@
     // plugin will fetch it from the file via the ::identify() method if data
     // is not already available.
     if (!$file_md = $this->fileMetadataManager->uri($this->getSource())) {
       // No file, return.
       return FALSE;
     }
+
+    if (\Drupal::state()->get("bos_migration.active", FALSE)) {
+      $migration_mode = \Drupal::state()->get("bos_migration.fileOps", "copy");
+      if ($migration_mode != "copy" && $migration_mode != "move") {
+        return FALSE;
+      }
+    }

     if (!$file_md->getMetadata(static::FILE_METADATA_PLUGIN_ID)) {
       // No data, return.
       return FALSE;
     }
