--- a/modules/contrib/imagemagick/src/Plugin/ImageToolkit/ImagemagickToolkit.php
+++ b/modules/contrib/imagemagick/src/Plugin/ImageToolkit/ImagemagickToolkit.php
@@ -732,3 +732,8 @@
-   * {@inheritdoc}
-   */
-  public function parseFile() {
+   * {@inheritdoc}
+   */
+  public function parseFile() {
+    $source = $this->getSource();
+    $ext = explode(".", $source);
+    if (end($ext) == "svg") {
+      return FALSE;
+    }
