--- a/modules/contrib/salesforce/modules/salesforce_jwt/src/Plugin/SalesforceAuthProvider/SalesforceJWTPlugin.php
+++ b/modules/contrib/salesforce/modules/salesforce_jwt/src/Plugin/SalesforceAuthProvider/SalesforceJWTPlugin.php
@@ -92,7 +92,7 @@
    * {@inheritdoc}
    */
   public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
-    if (!$this->keyRepository->getKeyNamesAsOptions(['type' => 'authentication'])) {
+    if (!$this->keyRepository->getKeyNamesAsOptions(['type' => ['authentication', 'asymmetric_private']])) {
       $this->messenger()->addError($this->t('Please <a href="@href">add an authentication key</a> before creating a JWT Auth provider.', ['@href' => Url::fromRoute('entity.key.add_form')->toString()]));
       return $form;
     }
@@ -126,7 +126,7 @@
       '#title' => 'Private Key',
       '#type' => 'select',
       '#empty_option' => $this->t('- Select -'),
-      '#options' => $this->keyRepository->getKeyNamesAsOptions(['type' => 'authentication']),
+      '#options' => $this->keyRepository->getKeyNamesAsOptions(['type' => ['authentication', 'asymmetric_private']]),
       '#required' => TRUE,
       '#default_value' => $this->getCredentials()->getKeyId(),
     ];
